src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.form-search {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
```

Comment:
Check CSS properties order. You can use this [link](https://9elements.com/css-rule-order/)

Reply:
On my second commit I double checked that I have no outline. But now I added outline:none everywhere to be sure.
<img width="710" alt="no outline" src="https://user-images.githubusercontent.com/104161452/200325941-0c88e393-56d9-44dd-9982-03eece347198.png">
<img width="709" alt="no outline2" src="https://user-images.githubusercontent.com/104161452/200325944-31ad48a5-9f7e-4300-a765-e3468bf0c2fb.png">